<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowPicsDlg</class>
 <widget class="QDialog" name="ShowPicsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片秀</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="m_frmImage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="m_spcLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnPrev">
       <property name="text">
        <string>上一张</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnNext">
       <property name="text">
        <string>下一张</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnClose">
       <property name="text">
        <string>关闭</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_btnPrev</tabstop>
  <tabstop>m_btnNext</tabstop>
  <tabstop>m_btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
